content/blog/2020-04-tidymodels-org/index.Rmarkdown

```diff
@@ -0,0 +1,51 @@
+---
+title: Announcing tidymodels.org
+slug: tidymodels-org
+description: >
+    We're thrilled to announce the launch of tidymodels.org! The site is a central location for learning and using the tidymodels packages. 
+date: 2020-04-17
```

Comment:
```suggestion
date: 2020-04-21
```

@batpigandme, @topepo, and anybody else (???) who we need to loop in: we would like to publish this post and launch the site next Tuesday, April 21.